frontend/public/components/nav.jsx

```diff
@@ -395,6 +395,7 @@ export class Nav extends React.Component {
             <ResourceNSLink resource="secrets" name="Secrets" onClick={this.close} />
             <ResourceNSLink resource="resourcequotas" name="Resource Quotas" onClick={this.close} />
             <ResourceNSLink resource="horizontalpodautoscalers" name="HPAs" onClick={this.close} />
+            <ResourceNSLink resource="serviceinstances" name="Service Instances" onClick={this.close} />
```

Comment:
We should only show service instances if service catalog is available. I'd add a flag to features.ts.

We should probably move this higher in the nav. I might put it just below the divider.